acs-apiclient/Droid/Activity/ExternalFlowWebActivity.cs

```diff
@@ -0,0 +1,130 @@
+﻿
+using System;
+using Android.App;
+using Android.OS;
+using Android.Views;
+using Android.Webkit;
+using acs_apiclient.Droid.CustomViews;
+using Android.Widget;
+using System.Threading;
+using AcsApi;
+using Android.Graphics;
+
+namespace acs_apiclient.Droid
+{
+    [Activity(Label = "ExternalFlowWebActivity", 
+    Theme = "@android:style/Theme.NoTitleBar", 
+    WindowSoftInputMode = SoftInput.AdjustResize)]
+    public class ExternalFlowWebActivity : Activity
+    {
+        public static string UrlParamKey = "url";
+        private static Android.Graphics.Color Obsidian = Android.Graphics.Color.ParseColor("#222b3c");
+        private static Android.Graphics.Color Ash = Android.Graphics.Color.ParseColor("#B7C0CC");
+        private static Android.Graphics.Color SelectedButtonTintColor = Ash;
+        private static Android.Graphics.Color UnselectedButtonTintColor = Obsidian;
+        private WebView contentWebView;
+        private String urlString;
+        private Android.Net.Uri uri;
+
+        protected override void OnCreate(Bundle savedInstanceState)
+        {
+            base.OnCreate(savedInstanceState);
+            this.SetContentView(Resource.Layout.activity_externalFlowWeb);
+            InitUri();
+            SetupTitle(uri.Host);
+            SetupCloseButton();
+            SetupSyncButton();
+            SetupWebView();
+        }
+
+        private void InitUri()
+        {
+            this.urlString = this.Intent.GetStringExtra(UrlParamKey);
+            if (String.IsNullOrEmpty(this.urlString))
+            {
+                throw new ArgumentException($"'url' must be provided in the Intent to start {typeof(ExternalFlowWebActivity)}");
```

Comment:
💡 This should reference ```UrlParamKey``` directly so it is updated if it changes

Reply:
Also the error could be more specific:
```$"A URL must be provided for the '{UrlParamKey} key in the Intent to start {typeof(ExternalFlowWebActivity)}"```

Reply:
Done

Reply:
✅